app/models/product.rb

```diff
@@ -1,4 +1,21 @@
 class Product < ApplicationRecord
+  PRODUCT_TAX_RATE = 0.08
+
+  has_one_attached :image do |attachable|
+    attachable.variant(:thumb, resize_to_fill: [300, 300])
+  end
+
   validates :name, presence: true
   validates :price, presence: true, numericality: { only_integer: true, greater_than: 0 }
+
+  scope :default_order, -> { order(position: :asc, created_at: :desc) }
```

Comment:
```suggestion
  scope :default_order, -> { order(position: :asc, id: :desc) }
```
厳密にはcreated_atも同じになり得るので、orderを不定にさせないためにidもいれときたい